packages/python-cryptography/build.sh

```diff
@@ -11,41 +11,25 @@ TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, python"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
-
-_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION)
+TERMUX_PYTHON_COMMOM_DEPS="wheel, cffi, setuptools-rust"
+TERMUX_PYTHON_TARGET_DEPS="cffi, cffi>=1.12"
```

Comment:
I believe pip requires quotes around versioned dependency, i.e. `cffi 'cffi>=1.12'` or `'cffi' 'cffi>=1.12'`

Reply:
Is it okay if in `TERMUX_PKG_PYTHON_TARGET_DEPS`, in `TERMUX_PKG_PYTHON_BUILD_DEPS` and in `TERMUX_PKG_PYTHON_COMMOM_DEPS` it will not be necessary to put commas for module separation?

Reply:
Probably best to keep comma, to be consistent with how TERMUX_PKG_DEPENDS is written